ibis/omniscidb/tests/conftest.py

```diff
@@ -36,26 +36,72 @@ def con():
     )
 
 
+class temp_helper:
```

Comment:
i get what you are trying to do. but i much prefer very explict operations in tests. I am ok with having a base class for some of this, but each operation should be a named function.

also classes need to be CamelCase generally.

Reply:
I'm not sure what do you mean under named function for each operation.

also changed class name to CamelCase style